server/routes/auth.routes.js

```diff
@@ -232,32 +335,44 @@ function userOnPasswordOnlyWhitelist(session){
  * @return {*|PromiseLike<T | never>|Promise<T | never>}
  */
 function createMAXUser(cas_data) {
```

Comment:
Also we can remove the "createMaxUser" and updateMaxUser functions since we do not use MAX anymore.

Just make sure to verify the login as Dev button still works when you remove.

Reply:
Just going to re-ping this since MAX.gov is not used anymore. Please let me know if we remove this createMaxUser and updateMaxUser if that will cause issues with the "Login as Dev" button